src/rez/package_maker__.py

```diff
@@ -128,6 +128,10 @@ def get_package(self):
     def _get_data(self):
         data = self._data.copy()
 
+        if "requires" in data:
+            if None in data["requires"]:
```

Comment:
You can do:
```python
data["requires"] = list(filter(None, data["requires"]))
```
Which is slightly more pythonic. Basically it will do one for loop instead of two.

Reply:
>Explicit is better than implicit.
Simple is better than complex.

Just joking, as a Haskeller I definitely approve but sometimes I find it hard to decide which way to go :)

Reply:
It's worth noting that `filter(None, ...)` will filter out anything that evaluates to `False`, not just `None` values.